edkrepo/common/clone_utilities.py

```diff
@@ -16,6 +16,7 @@
 import edkrepo.common.edkrepo_exception as edkrepo_exception
 import edkrepo.common.humble as humble
 import edkrepo.common.ui_functions as ui_functions
+import edkrepo.commands.humble.clone_humble as humble

```

Comment:
This file does not exist in the code base did you forget to include it in your commit? Additionally you cannot import two different files as humble.

Reply:
Removed the new humble file, and moved the string into common/humble.

```
#error messages for clone_utilities.py
CONFLICTING_PARTIAL_CLONE = 'Multiple partial clone arguments were provided.'
```